pkg_defs/ska3-flight/meta.yaml

```diff
@@ -42,15 +42,14 @@ requirements:
     - ska_parsecm ==4.0.0
     - ska_quatutil ==4.0.0
     - ska_shell ==4.0.1
-    - ska_sun ==3.12.0
+    - ska_sun ==3.13.0
     - ska_tdb ==4.0.0
     - ska3-core ==2023.3
```

Comment:
Doesn't this need an update to get the astropy thingy?

Reply:
Right, the file in HEAD /proj/sot/ska3/test there has the right date, but did you do that manually yesterday morning?  If so it would pass tests from that.
```
jeanconn-fido> grep expires /proj/sot/ska3/test/lib/python3.10/site-packages/astropy/utils/iers/data/Leap_Second.dat 
#  File expires on 28 June 2024
jeanconn-fido> ls -l /proj/sot/ska3/test/lib/python3.10/site-packages/astropy/utils/iers/data/Leap_Second.dat 
-rw-rw-r-- 2 aca aspect 1352 Jan 10 10:45 /proj/sot/ska3/test/lib/python3.10/site-packages/astropy/utils/iers/data/Leap_Second.dat
```

Reply:
Fixed it